x-pack/plugins/observability_solution/infra/public/pages/metrics/metrics_explorer/hooks/use_metrics_explorer_data.test.tsx

```diff
@@ -36,7 +36,7 @@ const queryClient = new QueryClient({
 });
 
 const renderUseMetricsExplorerDataHook = () => {
-  const wrapper: FC<PropsWithChildren> = ({ children }) => {
+  const wrapper: FC<PropsWithChildren<any>> = ({ children }) => {
```

Comment:
This was an `any` prior to the original PR as well, by committing the type for FC, this just makes it explicit. Would require a precise type.